staging/src/k8s.io/apiserver/pkg/endpoints/filters/audit_test.go

```diff
@@ -37,7 +37,7 @@ import (
 	"k8s.io/apimachinery/pkg/types"
 	"k8s.io/apimachinery/pkg/util/wait"
 	auditinternal "k8s.io/apiserver/pkg/apis/audit"
-	auditv1alpha1 "k8s.io/apiserver/pkg/apis/audit/v1alpha1"
+	auditv1beta1 "k8s.io/apiserver/pkg/apis/audit/v1beta1"
```

Comment:
Should tests that use versioned API live here? Maybe move them closer to the log backend, generalize and have both for beta and alpha?